rcw.py

```diff
@@ -811,7 +811,7 @@ def create_source():
     # After this stage, all the RCW bits should be formatted with lsb on
     # the right side and msb on the left side to permit conversion into
     # a very long uint.
-    bitstring = ''.join(['{0:08b}'.format(ord(x)) for x in bitbytes])[::-1]
+    bitstring = ''.join(['{0:08b}'.format(x) for x in bitbytes])[::-1]
```

Comment:
This fixed python3 issue:

```
File "/Users/davidgarske/GitHub/rcw/rcw.py", line 818, in <listcomp>
    bitstring = ''.join(['{0:08b}'.format(ord(x)) for x in bitbytes])[::-1]
                                          ^^^^^^
TypeError: ord() expected string of length 1, but int found
```

Thank you @sean-anderson-seco . How can this PR get merged?